apps/heureka/README.md

```diff
@@ -7,11 +7,30 @@ This mircro frontent uses the Heureka POC API which aims to:
 
 - Track the overall state of our technology landscape
 - Establish a unified & complete Patch Management Process:
+
   - Maintain & track vulnerabilities and affected components
   - Document the remediation, classification, and impact of vulnerabilities
   - Document the changes corresponding with patching of vulnerabilities, as well as updating components
 
-# Usage
+  **Coming soon:**
```

Comment:
What do you think about creating a separate "Coming Soon" section? The current layout looks a bit off to me. This is a good point to consider in the feature when you want to warn about things.

![Screenshot 2024-10-23 at 10 27 21](https://github.com/user-attachments/assets/051d1a3b-7d9d-44ac-a543-2798343f043c)